semantic error: append() takes exactly one argument
 --> tests/errors/test_list2.py:6:5
  |
6 |     a.append(4, 5)
  |     ^^^^^^^^^^^^^^ 
